uPortal-api/build.gradle

```diff
@@ -2,9 +2,4 @@
  * Skip installing and uploading this level of the hierarchy;  it's merely a convenient grouping
  * for submodules.
  */
-install {
-    enabled = false
-}
-uploadArchives {
-    enabled = false
-}
+tasks.withType(PublishToMavenRepository).configureEach { it.enabled = false }
```

Comment:
based off https://stackoverflow.com/questions/55579031/excluding-one-or-two-sub-projects-from-publication-in-gradle